scripts/New-ValidatedPR.ps1

```diff
@@ -0,0 +1,142 @@
+<#
+.SYNOPSIS
+  Creates a validated PR with all guardrails enforced.
+
+.DESCRIPTION
+  Wrapper around the New-PR skill that provides a convenient interface for
+  creating PRs with validation. This script delegates to the skill for better
+  cohesion and reusability.
+  
+  Exit codes:
+    0 = Success
+    1 = Validation failure  
+    2 = Usage/environment error
+
+.PARAMETER Title
+  PR title in conventional commit format (e.g., "feat: Add feature")
+
+.PARAMETER Body
+  PR description body
+
+.PARAMETER BodyFile
+  Path to file containing PR body
+
+.PARAMETER Base
+  Target branch (default: main)
+
+.PARAMETER Head
+  Source branch (default: current branch)
+
+.PARAMETER Draft
+  Create as draft PR
+
+.PARAMETER Web
+  Open browser to create PR interactively (bypasses validation)
+
+.PARAMETER SkipValidation
+  Skip validation checks. Requires -AuditReason. Use sparingly for emergencies only.
+
+.PARAMETER AuditReason
+  Required when -SkipValidation is used. Documents reason for skipping validation.
+
+.EXAMPLE
+  .\New-ValidatedPR.ps1 -Title "feat: Add feature" -Body "Description"
+  Creates PR after running all validations
+
+.EXAMPLE
+  .\New-ValidatedPR.ps1 -Title "feat: Feature" -BodyFile "pr-body.md" -Draft
+  Creates draft PR with body from file
+
+.EXAMPLE
+  .\New-ValidatedPR.ps1 -Web
+  Opens browser for interactive PR creation (no validation)
+  
+.EXAMPLE
+  .\New-ValidatedPR.ps1 -Title "fix: Hotfix" -SkipValidation -AuditReason "Production incident #123"
+  Creates PR without validation (audit logged)
+
+.NOTES
+  This is a thin wrapper around .claude/skills/github/scripts/pr/New-PR.ps1
+  which contains the core logic. The skill can be used directly for more control.
```

Comment:
@rjmurillo-bot It's unclear to me when we'd use this. Only for CI it looks like? Where are we keeping scripts that are only intended for CI? How are we organizing them? Is there a naming convention? Folder structure? If there isn't, create a standard and document that in an ADR. This is getting 🍌🍌🍌